### DISCUSSION TITLE
TeXstudio has been crashing a lot since version 3.0.4 and it remains in 3.1.0

### DISCUSSION THREAD

**Discussion by fernandozucatelli** [texstudio-org/texstudio#1495] [General]
I realized that TeXstudio has been crashing quite often recently. It is crashing almost once an hour, mainly when I navigate among the files through Ctrl + Click.

It is even more probable to crash when I open a new root file and try to compile it or navigate in its files.

Nowadays I'm closing TeXstudio and reopening it when I need to change the root file.

Note: I'm not sure if it should be a bug report at https://sourceforge.net/projects/texstudio/, since there is a sign saying the project is now on GitHub.

**Comment by sunderme** [comment]
Here is the right place to report.
Usually there is no need to change the root file manually.
I am not aware of an issue with 3.0.4, so please compile a minimal example with step-by-step instruction on how to crash txs.
